package/network/utils/comgt/files/ncm.sh

```diff
@@ -25,7 +25,7 @@ proto_ncm_init_config() {
 proto_ncm_setup() {
 	local interface="$1"
 
-	local manufacturer initialize setmode connect ifname devname devpath
+	local manufacturer initialize setmode connect finalize ifname devname devpath
```

Comment:
Where does `finalize` get used?  I don't see it in the diffs below... or was it already in use but undeclared?